packages/components/src/range-control/utils.ts

```diff
@@ -50,7 +50,10 @@ export function useControlledRangeValue(
 	const { min, max, value: valueProp, initial } = settings;
 	const [ state, setInternalState ] = useControlledState(
 		floatClamp( valueProp, min, max ),
-		{ initial, fallback: null }
+		{
+			initial: floatClamp( initial ?? null, min, max ),
```

Comment:
The `?? null` is to fix a type mismatch, since:
- `initial` can be `number | undefined`
- `floatClamp` accepts `number | null` as its first argument